index.html

```diff
@@ -76,16 +59,16 @@
 <body>
 <h1 class="title p-name" id="title">Requirements for Japanese Text Layout <br/><span lang="ja">日本語組版処理の要件（日本語版）</span></h1>
 
-<div id="abstract">
+<section id="abstract">
```

Comment:
If i remember correctly, we used the div markup here to avoid warning messages about using a section with no heading. You might want to check that out.